event-streams/jmxtrans/README.md

```diff
@@ -0,0 +1,180 @@
+# JmxTrans
+
+[JMX Trans](https://github.com/jmxtrans/jmxtrans) is a tool for collecting data from JMX endpoints of Java applications and sending it to other applications and services. 
+
+The ability to deploy JmxTrans was deprecated in Event Streams 11.1.5 and removed in Event Streams 11.2.0. While the JmxTrans tool remains unsupported, this document provides high-level guidance on how to deploy it separately alongside an Event Streams instance.
+
+## JmxTrans Configuration
```

Comment:
```suggestion
## Configuring JmxTrans by using the ConfigMap
```

Reply:
While we are using ConfigMap for this I don't think we need to say that in the title. I'm trying to keep the focus on what they user is trying to achieve, not how.

Reply:
Updated suggestion to be consistent with our docs. 😃